dashboard/dashboard.html

```diff
@@ -980,6 +980,7 @@
 	logSegment.appendChild(h("div", attrs, [
 		h('span', null, " IGNOR "),
 		h('a', {"href": data["url"], "className": "ignore"}, data["url"]),
+		data["source"],
```

Comment:
Yeah, you can use strings in the `h` children which will call `appendAny` and convert them to text nodes.

But `"(" + data["source"] + "): "` might as well be in the IGNOR span to reduce the amount of DOM.

I don't think there's any opportunity for XSS.